test/app_profiler/viewer/speedscope_remote_viewer/middleware_test.rb

```diff
@@ -60,6 +60,21 @@ class MiddlewareTest < TestCase
           assert_match(%r(<script type="text/javascript">), html)
         end
 
+        test "#call show can serve huge payloads" do
+          frames = { "1" => { name: "a" * 1e7 } }
```

Comment:
10MB, or about 10 million (`1e7`) characters, is the size limit the old HTML4 sanitizer wasn't able to exceed.
Reverting `rails-html-sanitizer` to 1.5.0 in Gemfile.lock, one can run this test and confirm that it fails.